src/TraceEvent/TraceLog.cs

```diff
@@ -7125,6 +7131,14 @@ internal TraceModuleFile UniversalMapping(ProcessMappingTraceData data, ProcessM
                 
```

Comment:
On line 7130, we set the timestamp to the start of the trace because mappings are enumerated at the start of the trace.  This doesn't always happen - we should set the timestamp properly.